### PR TITLE
[FEAT] Create panel for active buffs

### PR DESCRIPTION
**Pull request by joaofernandez**
# Description
! WIP !
Creates a section to view all active buffs. 

[Check this issue](https://github.com/sunflower-land/sunflower-land/issues/2480)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested 

image
_New menu icon_

image
_Skill Buffs List_

image
_Equips Buffs List_

image
_WIP Collectibles Buffs List_
* Could not find a good datasource to look for the image and the buffs descriptions of the collectibles.

image
_WIP Bud Buffs List_
* Tha same with the buds buffs and couldn't find a png bud image only a gif.

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/island/hud/components/buffs/CollectionBuffsList.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import chest from "assets/icons/chest.png";
+import { Label } from "components/ui/Label";
+import { OuterPanel } from "components/ui/Panel";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { getEntries } from "features/game/types/craftables";
+import { Collectibles } from "features/game/types/game";
+import { OPEN_SEA_ITEMS } from "metadata/metadata";
```
Comment (joaofernandez) [inline src/features/island/hud/components/buffs/CollectionBuffsList.tsx:7]: I found this data source that I used to obtain information about the collectibles. Is there a better way?

File: src/features/island/hud/components/inventory/Inventory.tsx [hunk 2]
```diff
@@ -102,6 +104,33 @@ export const Inventory: React.FC<Props> = ({
...
+          <img
+            src="/src/assets/skills/land/cooking/michelin_stars.png"
+            className="absolute"
+            style={{
+              top: `${PIXEL_SCALE * 3}px`,
+              left: `${PIXEL_SCALE * 5}px`,
+              width: `${PIXEL_SCALE * 12}px`,
+            }}
+          />
+        </div>
```
Comment (joaofernandez) [inline src/features/island/hud/components/inventory/Inventory.tsx:132]: Adds a new menu. Used this michelin_stars icon from the cooking skills.

File: src/features/island/hud/components/buffs/CollectionBuffsList.tsx (hunk 1 above)
Comment (elias7905) [reply src/features/island/hud/components/buffs/CollectionBuffsList.tsx:7]: Hey @joaofernandez first of all, amazing work you've done here!
maybe you can check "src\features\game\types\collectibles.ts"?

**Comment by spencerdezartsmith** [comment]
This is an awesome initiative mate! I do want to mention that we will be building this functionality/UI for our next season. We can take some inspiration from what you have here but I would hate for you to put in a ton of time without knowing that we will be building this ❤️

**Comment by joaofernandez** [comment]
Thank you @spencerdezartsmith! Not a problem at all, I used this feature to learn more about the project :)

**Pull request closed without merging**